addons/product/models/product_template.py

```diff
@@ -48,6 +49,14 @@ def _read_group_categ_id(self, categories, domain, order):
         'Sales Description', translate=True,
         help="A description of the Product that you want to communicate to your customers. "
              "This description will be copied to every Sales Order, Delivery Order and Customer Invoice/Credit Note")
+    description_ecommerce = fields.Html(
+        string="eCommerce Description",
+        translate=html_translate,
+        sanitize_overridable=True,
+        sanitize_attributes=False,
+        sanitize_form=False,
+    )
+    show_description_ecommerce = fields.Boolean(compute='_compute_show_description_ecommerce')
```

Comment:
Why this other field + 2 methods `_compute_show_description_ecommerce` and `_get_show_description_ecommerce`?

Can't you just set the field as `invisible` by default in `product` and add an xpath in both `website_sale` and `post_self_order` which would set the `invisible` attribute to `0`?

Since the condition to show it is "as soon as there is one module installed, show it" and since there is no notion of "this module ask to show it but this one ask to hide it, we need a strong business logic", seems like it would work in a more straightforward / simple way with 2 simple xpath.